src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java

```diff
@@ -1,6 +1,6 @@
 /*
  * SonarQube :: Packaging Maven Plugin
- * Copyright (C) 2009-2017 SonarSource SA
+ * Copyright (C) 2009-2018 SonarSource SA
```

Comment:
Why not 2019?

Reply:
good question, maybe parent pom is not the most recent one, will check.